docs/okeps/okep-5094-layer2-transit-router.md

```
@@ -0,0 +1,895 @@
+
+The pod to remote pod traffic is not affected since the logical switch will
+keep intact the pod's local and remote LSPs with its tunnel key.
+
+#### Egress traffic at local node
+
+At normal egress without an egress IP, traffic exits through the node
+hosting the pod/VM. Therefore, a partial cluster upgrade does not impact
+the logical network topology.
+
+However, the pod/VM will continue to use outdated gateway values:
+- **IPv4**: Gateway router LRP MAC address.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)